process/throttle/antiflood/floodPreventers/quotaFloodPreventer.go

```diff
@@ -25,60 +38,75 @@ type quota struct {
 
 // quotaFloodPreventer represents a cache of quotas per peer used in antiflooding mechanism
 type quotaFloodPreventer struct {
-	maxMessagesPerPeer uint32
-	mutOperation       *sync.RWMutex
-	cacher             storage.Cacher
-	statusHandlers     []QuotaStatusHandler
-	maxSizePerPeer     uint64
-	percentReserved    uint32
+	name                          string
+	mutOperation                  *sync.RWMutex
```

Comment:
is the pointer to the RWMutex necessary?

Reply:
no, it was necessary in the old implementation in which the p2p antiflood struct was constructed above quotaFloodPreventer. Will remove the pointer ASAP.

Reply:
done